<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VkontakteSettingsPage</class>
 <widget class="QWidget" name="VkontakteSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vkontakte</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="LoginStateWidget" name="login_state" native="true"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Account details</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QWidget" name="login_container" native="true">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout" columnstretch="1,0">
         <property name="margin">
          <number>0</number>
         </property>
         <item row="2" column="1">
          <widget class="QPushButton" name="login">
           <property name="text">
            <string>Login</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="details_label">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Not logged in</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LoginStateWidget</class>
   <extends>QWidget</extends>
   <header>widgets/loginstatewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
